students/eowyn/session05/mailroom.py

```diff
@@ -0,0 +1,161 @@
+#!/usr/bin/env python
+
+import sys
+
+''' global variables'''
+donor_names = ["Margaret Atwood", "Fred Armisen",
+               "Heinz the Baron Krauss von Espy"]
+donations = [[300, 555], [240, 422, 1000], [1500, 2300]]
+DONORS = dict(zip(donor_names, donations))
+
+
+
+def safe_input():
+    return None
+
+def quit_code():
+    sys.exit()
+
+def return_to_menu():
+    ''' Raise a zero division error to trigger exit out of loop'''
+    return 5/0
+
+def list_donors():
+    ''' List all DONORS '''
+    print("All Donors:")
+    [print(x) for x in DONORS]
+
+
+def generate_table():
+    ''' Pretty-print a table of donor statistics '''
+    [name, total, number, ave] = [[], [], [], []]
```

Comment:
no need for the square brackets here:

` name, total, number, ave = [], [], [], []`

does the same thing, slightly more efficiently.